tensorflow/python/util/BUILD

```diff
@@ -76,6 +76,7 @@ tf_python_pybind_extension(
         "//tensorflow/python:pybind11_lib",
         "//third_party/python_runtime:headers",
         "@pybind11",
+        "//tensorflow/core/platform:platform_port",
```

Comment:
The list is sorted alphabetically. Could you please move this line to before `//tensorflow/python:pybind11_lib",`? Thank you very much!

Reply:
thank you for pointing out. Let me do this now.

Reply:
I fixed that. If you find any more required fix, please let me know :)